### PR TITLE
Update Qubes Alpha Install Instructions

### PR DESCRIPTION
**Pull request by deeplow**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -117,24 +117,32 @@ Create a .rpm:
...
 
-#### In `dom0`
+Overview of the qubes you'll create:
 
-1. Create a new Fedora **template** (`fedora-37-dz`) for Dangerzone development:
+>    | qube         |   type   | purpose |
+>    |--------------|----------|---------|
+>    | dz           | app qube | Dangerzone development |
+>    | dz-dvm       | app qube | offline diposable template for performing conversions
+>    | fedora-38-dz | template | template for the other two qubes
```
Comment (apyrgio) [inline BUILD.md:126]: Two quick comments here:

1. I think we can present the table without the block quotes.
2. You've left some `|` markers out of the last two lines (see end of line). GitHub still renders the table, but we could be more strict.

File: BUILD.md [hunk 2]
```diff
@@ -180,6 +188,7 @@ specified qubes.
 
    ```
    git clone https://github.com/freedomofpress/dangerzone
+   cd dangerzone
```
[the remark this answers is not in the document]
Comment (deeplow) [reply BUILD.md:191]: Good point. I had missed that. Should be fixed by 1a7aea8 .

File: BUILD.md (hunk 1 above)
Comment (deeplow) [reply BUILD.md]: Done in [af1ffac](https://github.com/freedomofpress/dangerzone/pull/533/commits/af1ffacc5838fdc37986a6fb0ef2a561c8ed8017)

**Comment by apyrgio** [comment]
Thanks for the changes. I resolved the conversations, feel free to merge.

**Pull request merged**